### ISSUE TITLE
loss

### ISSUE THREAD

**Issue by abbasrazaali** [kwotsin/transfer_learning_tutorial#15]
hi,
are you using loss, it isn't used after declaration ?
loss = tf.losses.softmax_cross_entropy(onehot_labels = one_hot_labels, logits = logits)

**Comment by kwotsin** [comment]
If the code is not using any loss, the model cannot be trained in the conventional sense. could you explain what you mean by "after declaration"?

**Comment by kwotsin** [comment]
Closing due to inactivity.
